packages/theme/components/FiltersSidebar.vue

```diff
@@ -123,6 +159,39 @@ export default {
       }), {});
     };
 
+    const setStartingRange = ref(() => {
+      if (typeof selectedPrice.value !== 'string') {
+        [0, 300];
+      } else {
+        const splittedPriceString = selectedPrice.value.split(',');
+        [splittedPriceString[0], splittedPriceString[1]];
+      }
+    });
+
+    const priceRangeConfig = ref({
+      start: range,
+      range: {min: 0, max: 300},
+      step: 1,
+      connect: true,
+      direction: 'ltr',
+      orientation: 'horizontal',
+      behaviour: 'tap-drag',
+      tooltips: true,
+      keyboardSupport: true,
+      format: {to: function(value) {
+        return new Intl.NumberFormat([locale.value, locale.value.toUpperCase()].join('-'), {style: 'currency', currency: localCurrency.value}).format(value);
+      },
+      from: function(value){
+        return value;
+      }}
+    });
+
```

Comment:
We should do parsing of a string here instead than in a separate function. This way the component will have a consistent interface for us (we put number in it, we get numbers out of it)